lib/filter_lib/src/pagination.py

```diff
@@ -35,11 +59,18 @@ def _calculate_num_pages(page_size: int, total_results: int) -> int:
 
 
 def paginate(items: Sequence[T], pag_params: PaginationParams) -> Page[T]:
-    pag = PaginationOut(
-        page_num=pag_params.page_num,
-        page_size=pag_params.page_size,
-        min_pages_left=pag_params.min_pages_left,
-        total=pag_params.total,
-        has_more=pag_params.has_more,
-    )
+    pagination_out_attrs = {
```

Comment:
Use dict comprehension to build params for PaginationOut

Reply:
corrected with using dict comprehension